### PR TITLE
Add translation for DDoc version on About page

### PR DESCRIPTION
**Pull request by alxndrsn**
Issue: medic/medic-webapp#2084

### CODE REVIEW

File: translations/messages-en.properties [hunk 1]
```diff
@@ -577,6 +577,7 @@ debug.db_info.title = Database Info
 debug.mode.title = Enable debug mode
 debug.mode.description = Debug mode will print information into the browser console to help developers diagnose problems with the application.  After changing this setting reload the app so it applies.
 android_app.version.title = Android App Version
+ddoc.version = DDoc Version
```
Comment (garethbowen) [inline translations/messages-en.properties]: Maybe change the value to something a little more user friendly. What about "Application Version"?

Comment (abbyad) [inline translations/messages-en.properties]: "Application Version" may be confounded with the "webapp version", or the "android app version". Does something like "Config version" work?

Comment (garethbowen) [inline translations/messages-en.properties]: It's both the "webapp version" and the "config version" all in one magic number. I'm not sure how to express that in English...

Comment (alxndrsn) [inline translations/messages-en.properties]: Possibly safer to stick with ddoc, then?

**Pull request closed without merging**